CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.0.27
```

Comment:
Can this version bump be the next `-dev` iteration rather than a new version? This lets us control what gets bundled into a single release.